.github/workflows/build-and-publish.yml

```diff
@@ -95,24 +96,17 @@ jobs:
             **/gradle/**
             Scripts/**
   
-  githubRelease:
+  create_Tag:
     needs: maven_Release
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
out of curiosity, why windows here?

Reply:
The powershell script runs differently on ubuntu which leads to a null value exception.

Reply:
https://github.com/microsoftgraph/msgraph-sdk-java-core/runs/3613790056?check_suite_focus=true

Reply:
fixed the script.